src/daemon/main.cpp

```diff
@@ -186,13 +184,6 @@ int main(int argc, char const * argv[])
       return 0;
     }
 
-    // OS
-    if (command_line::get_arg(vm, daemon_args::arg_os_version))
-    {
-      std::cout << "OS: " << tools::get_os_version_string() << ENDL;
```

Comment:
You should also remove all the cruft in `common/util.h` related to this: the `get_os_version_string()` function plus the 200 lines of code to figure out whether you are running on `Windows 7 Enterprise Edition (core installation)` or `Windows Server 2012 R2 Web Server Edition` or a thousand other Windows versions.

I know, it's sad: Windows users will have to use something other than `lokid` to figure out which version of Windows they are using!

Reply:
Actually don't worry about going this, I've got it ripped out in a commit anyway that I'll sort out when I merge RPC with this PR.